docs/next.config.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
             oneOf: [
               {
                 resourceQuery: /@mui\/markdown/,
-                use: require.resolve('@mui/markdown/loader'),
+                use: [options.defaultLoaders.babel, require.resolve('@mui/markdown/loader')],
```

Comment:
@Janpot Trivial clarification, but wanted to ask why there is the need to chain the  `options.defaultLoaders.babel` here?